packages/cfd/src/Components/cfd-poi.tsx

```diff
@@ -26,7 +26,7 @@ type TFormValues =
           tax_identification_number?: string;
           account_opening_reason?: string;
       }
-    | {};
```

Comment:
it gave a ESLint/TS warning: 'Don't use `{}` as a type. `{}` actually means "any non-nullish value".
- If you want a type meaning "any object", you probably want `Record<string, unknown>` instead.
- If you want a type meaning "any value", you probably want `unknown` instead.
- If you want a type meaning "empty object", you probably want `Record<string, never>` instead.'